ui/packages/app/src/ApiDocs.svelte

```diff
@@ -16,99 +16,121 @@
+									{((label) => {
+										return label ? "'" + label + "'" : "the";
+									})(
+										components.filter((c) => c.id === dependency.inputs[i])[0]
+											.props.label
+									)}
 
-								<span class="capitalize"
-									>{components.filter((c) => c.id === dependency.inputs[i])[0]
-										.props.name}</span
-								> component
-							</span>
-							<br />
-						{/each}
-						&nbsp;&nbsp;]<br />
-						&#125;
-					</div>
-					<h4 class="text-2xl mt-6 mb-4">Response Object</h4>
-					<div
-						class="block mb-4 text-lg bg-gray-100 dark:bg-gray-700 p-4 font-mono"
-					>
-						&#123;<br />
-						&nbsp;&nbsp;"data": [<br />
-						{#each dependency.documentation[1] as dependency_doc, i}
-							&nbsp;&nbsp;&nbsp;&nbsp;{dependency_doc[1]},
-							<span class="text-pink-400 dark:text-pink-600"
-								>// represents {dependency_doc[0]} of
-								{((label) => {
-									return label ? "'" + label + "'" : "the";
-								})(
-									components.filter((c) => c.id === dependency.outputs[i])[0]
-										.props.label
-								)}
-								<span class="capitalize"
-									>{components.filter((c) => c.id === dependency.outputs[i])[0]
-										.props.name}</span
-								> component
-							</span>
-							<br />
-						{/each}
-						&nbsp;&nbsp;],<br />
-						&nbsp;&nbsp;"duration": (float)
-						<span class="text-pink-400 dark:text-pink-600">
-							// number of seconds to run function call</span
-						><br />
-						&#125;
+									<span class="capitalize"
+										>{components.filter((c) => c.id === dependency.inputs[i])[0]
+											.props.name}</span
+									> component
+								</span>
+								<br />
+							{/each}
+							&nbsp;&nbsp;]<br />
+							&#125;
+						</div>
+						<h4 class="text-2xl mt-6 mb-4">Response Object</h4>
+						<div
+							class="block mb-4 text-lg bg-gray-100 dark:bg-gray-700 p-4 font-mono"
+						>
+							&#123;<br />
+							&nbsp;&nbsp;"data": [<br />
+							{#each dependency.documentation[1] as dependency_doc, i}
+								&nbsp;&nbsp;&nbsp;&nbsp;{dependency_doc[1]},
+								<span class="text-pink-400 dark:text-pink-600"
+									>// represents {dependency_doc[0]} of
+									{((label) => {
+										return label ? "'" + label + "'" : "the";
+									})(
+										components.filter((c) => c.id === dependency.outputs[i])[0]
+											.props.label
+									)}
+									<span class="capitalize"
+										>{components.filter(
+											(c) => c.id === dependency.outputs[i]
+										)[0].props.name}</span
+									> component
+								</span>
+								<br />
+							{/each}
+							&nbsp;&nbsp;],<br />
+							&nbsp;&nbsp;"duration": (float)
+							<span class="text-pink-400 dark:text-pink-600">
+								// number of seconds to run function call</span
+							><br />
+							&#125;
+						</div>
 					</div>
-				</div>
-			{/if}
-		{/each}
-	</div>
+				{/if}
+			{/each}
+		</div>
+	{:else}
+		<h2 class="text-3xl text-center mb-6">
```

Comment:
Big diff but I just added an if else, where the else clause is the case of no named api routes.